packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Graph2dRangeXTask/Graph2dRangeXTask.js

```diff
@@ -0,0 +1,62 @@
+import { Markdown, Text } from 'grommet'
+import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
+import { observable } from 'mobx'
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+function storeMapper (stores) {
+  const {
+    addAnnotation
+  } = stores.classifierStore.classifications
+  const annotations = stores.classifierStore.classifications.currentAnnotations
+  return {
+    addAnnotation,
+    annotations
+  }
+}
+
+@inject(storeMapper)
+@observer
+class Graph2dRangeXTask extends React.Component {
+  onChange (index, event) {
+    const { addAnnotation, task } = this.props
+    if (event.target.checked) addAnnotation(index, task)
+  }
+
+  render () {
+    const {
+      annotations,
+      task
+    } = this.props
+    let annotation
```

Comment:
Is this being used in the render method at all?

Reply:
It should be - there's supposed to be a "X marks made" counter/button on the Task area. I've added it in, in the latest commit in this PR, to make this prop less dangly.

![screen shot 2019-01-22 at 13 40 45](https://user-images.githubusercontent.com/13952701/51539210-593c1100-1e4b-11e9-9ae1-8e132990d13d.png)
_What the Task Area is supposed to ultimately mimic_